src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -140,15 +147,27 @@ const NavBar: React.FC<NavBarPropsType> = ({
   }, [radioFirstPlayStateLoaded]);
 
   const [isEventScheduleVisible, setEventScheduleVisible] = useState(false);
-  const toggleEventSchedule = useCallback(() => {
+  function toggleEventSchedule() {
+    console.log("toggling");
     setEventScheduleVisible(!isEventScheduleVisible);
-  }, [isEventScheduleVisible]);
+  }
```

Comment:
`console.log` will need to be removed from here.

We also use `const foo = () => {` style for our functions, rather than `function foo() {`

Also, the `useCallback` wrapper seems to have been lost here, which is required to be kept to ensure the function maintains a stable reference, rather than being re-created on every render.

Reply:
I have  removed `log`, and converted functions to desired style.  @soichih - I would need your help on the `useCallback` part since I believe it was part of a solution to get schedule toggable by shortcut

Reply:
Sounds good. Is the `useCallback` aspect here still an issue? Or is this outdated now based on newer patterns?

Reply:
I believe it is no longer pertinent as we reverted to use mousetrap after you pointed to the real issue of overlapping shortcuts

Reply:
Cool, makes sense :) Mostly just wanted to ensure it was 'covered' now that i'm catching up on the comments/changes here.